app/main.py

```diff
@@ -1,2 +1,101 @@
+from typing import Any, List, Hashable
+
+
+class Node:
+    def __init__(self, key: Any, hash_value: int, value: Any) -> None:
```

Comment:
`key` must be `Hashable`
```suggestion
    def __init__(self, key: Hashable, hash_value: int, value: Any) -> None:
```

Reply:
thank you!